library/Fission/Web.hs

```diff
@@ -67,20 +64,17 @@ app = do
 
 -- | Construct an authorization context
 mkAuth ::
-  ( Has Heroku.ID       cfg
-  , Has Heroku.Password cfg
-  , HasLogFunc          cfg
+  ( HasLogFunc          cfg
   , MonadReader         cfg m
   , MonadDB        (RIO cfg)
+  , MonadHerokuAddOn m
   )
   => m (Context Auth.Checks)
 mkAuth = do
-  Heroku.ID       hkuID   <- Config.get
-  Heroku.Password hkuPass <- Config.get
-  cfg <- ask
-
+  cfg        <- ask
+  herokuAuth <- authorize
```

Comment:
:nail_care: 
I think this might be a little more clear as 
```
herokuAuth <- Heroku.authorize
```
the name `herokuAuth` helps, but it's still not immediately apparent where `authorize` is coming from